mobile/native/src/api_model/order/mod.rs

```diff
@@ -0,0 +1,111 @@
+use crate::common::ContractSymbol;
+use crate::common::Direction;
+use crate::trade;
+use flutter_rust_bridge::frb;
+
+pub mod notifications;
+
+#[frb]
```

Comment:
Do you think it makes sense if we can keep flutter related types isolated somehwere?